IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Entities/IfcRelInterferesElements/Documentation.md

```diff
@@ -1,12 +1,25 @@
-The _IfcRelInterferesElements_ objectified relationship indicates that two elements interfere. Interference is a spatial overlap between the two elements. It is a 1 to 1 relationship. The concept of two elements interfering physically or logically is described independently from the elements. The interference may be related to the shape representation of the entities by providing an interference geometry.  
+The _IfcRelInterferesElements_ objectified relationship indicates that two elements interfere. Interference is a spatial and/or shape interaction between the two elements. It is a 1 to 1 relationship. The concept of two elements interfering physically or logically is described independently from the elements. The interference may be related to the shape representation of the entities by providing an interference geometry.  
```

Comment:
What is a spatial interaction and how is it different from a shape interaction?

Reply:
To me spatial interaction state that footprint associated with SpatialElement have an interaction, in link with the change of paradigm that SpatialElement does not have proper shapes (intrinsic geometrical representation) but only footprint (documentation-related and communication-related area/spaces representation).

Reply:
Can we bring this logic somehow into the documentation? Otherwise, it is not specified.

Reply:
Added in the documentation:

The interference may be related to the shape representation of the entities by providing an interference geometry or zone:  
\* when the interference is between two physical products, the \_InterferenceGeometry\_ attribute is used to define the physical interference shape, it can be part of the shape of one of the elements used in the relationship or define the interface between the two shapes using a \_IfcConnectionGeometry\_.  
\* when the interference is between two spatial objects, the \_InterferenceSpace\_ attribute is used to define the interference space between the two associated footprints, expressed by a specific \_IfcSpatialZone\_ of predefined type \_IfcSpatialZoneTypeEnum\_ INTERFERENCE.

Reply:
what happens if the _RelatingElement_ is a spatial element, and the _RelatedElement_ is a physical element? Are such combinations allowed at all? Also is it legal to have _InterferenceGeometry_ AND _Interference_Space_ (both relations instantiated)?

Reply:
@TLiebich   
This is not explicitly forbidden, but the documentation express only clear homogeneous usage.  
IMHO cross usage need a more deep review of this relation -> **IFC Tunnel 4.4** 

Both attribute should not be set together:

*   Documented on line (2 lines below)
*   @SergejMuhic will add a WHERE clause to forbid this possibility